internal/gw/client_publish_errors_test.go

```diff
@@ -106,6 +106,19 @@ func TestClientPublishErrors(t *testing.T) {
 		}
 	})
 
+	t.Run("todo", func(t *testing.T) {
```

Comment:
Why "todo" ?

Reply:
Oops, forgot to update this. Re-pushed with updating subtest name.